packages/fast-components-react-msft/src/subheading/examples.data.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { IComponentFactoryExample  } from "@microsoft/fast-development-site-react";
+import Subheading from "./index";
+import { 
+    ISubheadingHandledProps,
+    ISubheadingUnhandledProps,
+    SubheadingTag,
+    SubheadingLevel
+} from "./subheading.props";
+
+import schema from "./subheading.schema.json";

```

Comment:
@janechu @nicholasrice what is our code guidance around naming import statements?  In this example I see camelCasing and Pascal?  Which should be used for consistency?